### ISSUE TITLE
moderation screen: deadlock detected warning upon accepting a post

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-infrastructure/answers.ros.org#149]
Just leaving this here: upon accepting a posting by a new user I was shown the following in a yellow 'box' at the top of the page:

```
deadlock detected DETAIL: Process 24052 waits for ShareLock on transaction 136307822; blocked by process 24051. Process 24051 waits for ShareLock on transaction 136307823; blocked by process 24052. HINT: See server log for query details
```

Seems to have recovered itself (or it's still hanging): the site works.

**Comment by gavanderhoorn** [comment]
Probably no longer relevant.
